project-handbook/technical-infrastructure.md

```diff
@@ -0,0 +1,63 @@
+# Overview of OpenRail's Technical Infrastructure
+
+The technical infrastructure of the OpenRail Association is comprised of various components to enable the work of the OpenRail projects as well as of different working groups such as the *OpenRail Team*.
+
+For the composition, we follow these principles:
+
+* We prefer Open Source
+* We chose software and services based on whether they are feasible and pragmatic
+* We prefer using hosted services instead of self-hosting
```

Comment:
It might be useful to explain why (also, are those points ordered?)

Reply:
Thanks. I added a sentence that may somewhat explain the ordering - they are not, it's `<ul>` ;). Regarding explanation, I could have probably written at least 3 paragraphs about each of these points but I don't believe that'd be overly useful as we may discuss each case individually. It's not a law, it's a snapshot of the guidelines we applied to far.